data/nation/liege.ruleset

```diff
@@ -0,0 +1,78 @@
+[nation_liege]
+
+name=_("Liegean")
+plural=_("?plural:Liegeans")
+groups="Modern", "European"
+legend = _("Liege was constituted in ecclesiastic principalty in 985,\
+ when Bishop Notger was granted the control of several cities in current Belgium.\
+ As a federated state of the Holy Roman Empire, it was characterized\
+ with large civil liberties for which liegean struggled strongly for.\
```

Comment:
I am not sure it's the right demonym

Reply:
I can't find it on the interwebs, and "liegian" is used for somewhere else.